fastify.js

```diff
@@ -6,7 +6,7 @@ const http = require('http')
 const https = require('https')
 const pinoHttp = require('pino-http')
 const Middie = require('middie')
-const fastseries = require('fastseries')
+const runHooks = require('fastseries')()
```

Comment:
I can, but we'll have the same problem [here](https://github.com/fastify/fastify/blob/master/lib/handleRequest.js#L5).